xsl/webhelp-common.xsl

```diff
@@ -525,6 +525,15 @@ border: none; background: none; font-weight: none; color: none; }
                 </div>
 
                 <xsl:call-template name="user.footer.navigation"/>
+                <xsl:choose><xsl:when test="$webhelp.include.search.tab = '0'">
```

Comment:
You can ignore this, I know see that this variable does the opposite of what I thought.